hamilton/parameter_analysis/plot_correlations.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+
+import pandas as pd
+import seaborn as sns
+
+
+if __name__ == "__main__":
+    dir = sys.argv[1]
+    filename = sys.argv[2]
+    file_path = os.path.join(dir, filename)
+    df = pd.read_csv(file_path)
+    y_vars = ["mean_squared_error"]
+
+    if filename.endswith("best_mses.csv"):
+        x_vars = list(df.columns.values)[0:-3]
```

Comment:
I can't remember what these numbers refer to. Would it make sense to assign them to global variables?

Reply:
I've just added comments to explain the magic numbers; I think it's probably easier to understand in context than moving to globals out of context.